src/ios/getPlistPath.js

```diff
@@ -6,7 +6,11 @@
  * Returns `null` if INFOPLIST_FILE is not specified.
  */
 module.exports = function getPlistPath(project) {
-  const plistFile = project.getBuildProperty('INFOPLIST_FILE');
+  const target = project.getFirstTarget().firstTarget;
+  const config = project.pbxXCConfigurationList()[target.buildConfigurationList];
+  const buildSection = project.pbxXCBuildConfigurationSection()[config.buildConfigurations[0].value];
+
+  const plistFile = buildSection.buildSettings.INFOPLIST_FILE;
```

Comment:
I have to test it tomorrow, but the CocoaPods were broken because of:
https://github.com/alunny/node-xcode/blob/master/lib/pbxProject.js#L1765

When value matching is found, there's no break, which means if there's another build configuration section w/o plist property (added by CocoaPods), it will overwrite Info.plist we found earlier.

This fixes that by not using this method and directly reading build values from the main target of the app. Just need to unit test & improve the general style here.